charts/chart-updater/values.yaml

```diff
@@ -20,9 +20,9 @@ git:
   # Path within git repo to locate Kubernetes manifests (relative path)
   path: ""
   # Username to use as git committer
-  user: "Chart Sync"
+  user: "Chart Updater"
   # Email to use as git committer
-  email: "chart-sync@rossum.ai"
+  email: "chart-updater@rossum.ai"
```

Comment:
Didn't we talk about using the rossum.cloud domain?

Reply:
Fixed, thx!